model/Export/ApipPackageExportHandler.php

```diff
@@ -57,25 +62,28 @@ public function getLabel()
 
     /**
      * @param core_kernel_classes_Resource $resource
-     * @return \tao_helpers_form_Form
+     *
+     * @return tao_helpers_form_Form
+     * @throws common_Exception
+     * @throws common_exception_Error
      */
     public function getExportForm(core_kernel_classes_Resource $resource)
     {
         if ($resource instanceof core_kernel_classes_Class) {
-            $formData = ['class' => $resource];
+            $formData['items'] = $this->getResourceService()->getAllChildren($resource);
```

Comment:
Not only the form should exclude the items, but the export request handler itself should filter out inaccessible resources.
Right now, I can easily download items, I don't see on that form by calling `/taoItems/ItemExport/index`, extending the list of `instances` manually.

Reply:
fixed

Reply:
Right, my bad, misplaced the original comment.
Good job!